manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/metadata.yaml

```diff
@@ -34,19 +34,27 @@ spec:
         app.kubernetes.io/name: {{ .Release.Name }}
     spec:
       containers:
-        - command:
-            - ./server/server
-            - --http_port=8080
-            - --mysql_service_host=mysql
-            - --mysql_service_port=3306
-            {{ if .Values.managedstorage.databaseNamePrefix }}
-            - --mlmd_db_name={{ .Values.managedstorage.databaseNamePrefix }}_metadata
-            {{ else }}
-            - --mlmd_db_name={{ .Release.Name | replace "-" "_" | replace "." "_" | trunc 50 }}_metadata
-            {{ end }}
-
-          image: {{ .Values.images.metadataserver }}
-          name: container
-          ports:
-            - containerPort: 8080
-              name: md-backendapi
+      - name: container
+        image: {{ .Values.images.metadataserver }}
+        env:
+        - name: DBCONFIG_USER
+            valueFrom:
+              secretKeyRef:
+                name: mysql-credential
+                key: username
+        - name: DBCONFIG_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: mysql-credential
+                key: password
+        command: ["/bin/metadata_store_server"]
+        args: ["--grpc_port=8080",
+                 "--mysql_config_host=mysql",
+                 "--mysql_config_database=metadb",
```

Comment:
could you revert this change? the db name need to be parameterized

            {{ if .Values.managedstorage.databaseNamePrefix }}
            - --mlmd_db_name={{ .Values.managedstorage.databaseNamePrefix }}_metadata
            {{ else }}
            - --mlmd_db_name={{ .Release.Name | replace "-" "_" | replace "." "_" | trunc 50 }}_metadata
            {{ end }}

Reply:
@neuromage - This should be fine to use a custom database name correct? I remember you asking if we named the database as "metadb", not sure you asked because there is a dependency on the name somewhere or was it just for consistency.

Reply:
As per discussion, updated it.

@IronPan Is the syntax correct? I changed it add quotes, but let me know if its correct or needs modification.

Reply:
Syntax LGTM
Remove "--mysql_config_database=metadb" ?

Reply:
Arrg, my bad. Fixed it.